.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI ARC E2E Linux VM Test
+
+on:
+  workflow_dispatch:
+  
+jobs:
+  setup-steps:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+      - name: Install latest version of Kind
+        run: go install sigs.k8s.io/kind@latest
+      - name: Create Kind cluster
+        run: |
+          PATH=$(go env GOPATH)/bin:$PATH
+          kind create cluster --name e2e-test
+      - name: Get Token
+        id: get_workflow_token
+        uses: peter-murray/workflow-application-token-action@v1
```

Comment:
I would pin this to a specific SHA instead of using v1. Tags in GitHub are not immutable (can be deleted / re-created), a fix is on the way but until then it's best to pin to a specific version

Reply:
There are pros and cons to both but I'm ok to add the SHA as well for now, updated